### DISCUSSION TITLE
[Feature] publish docker images for postgres and redis with :release tag

### DISCUSSION THREAD

**Discussion by williamkray** [immich-app/immich#12441] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

i deploy the immich stack in conjunction with many other services i run using docker compose. as such, i do not simply download the latest docker-compose file and run it, but rather have to analyze its contents, and see how the changes map to my personalized docker-compose.yml file.

this is generally not a problem as there aren't significant changes generally, but the most notable hangup is that postgres and redis containers are using upstream images that aren't tied to a specific label, but an image hash, which i have to compare strings against to see if i need to update which container to run for the latest version of immich.

it would be awesome (and relatively trivial to accomplish) for the immich dev team to tag the specific versions of those images and push them to their registry as `immich-postgresql:release` etc, so that users can continue to leverage the `:release` flag for all containers, and the release can be tied to the version of immich that requires and has been tested against that specific version of the container image.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
These are projects external to Immich and it would not be suitable for us to version them, nor is it necessary. Updates to those components shouldn't happen in tandem with Immich releases but just when they have their own new versions (and for postgres, only with care). 

Note that those upstream images _are_ tied to a label in our default compose file, just followed by a hash as well to pin to a specific version of that label for consistency. You're free to not use that hash in your own setup if you prefer, of course.
